roles/confluent.kafka_broker/handlers/main.yml

```diff
@@ -1,3 +1,3 @@
 ---
 - name: restart kafka
-  include_tasks: roles/confluent.kafka_broker/tasks/restart_kafka.yml
```

Comment:
I think this has the relative path from the base of the dir because the upgrade playbooks rely on this. 

I wonder if there is a variable that can be used

Or if jinja could help like

```
include_tasks: "{% if running_upgrade|bool %}roles/confluent.kafka_broker/{% endif %}tasks/restart_kafka.yml"
```

Reply:
@domenicbove Is there any way to fix the upgrade playbooks? To me it seems odd to have an "external" path to a role-internal file.

Reply:
I think that ^^ should work actually, we would just need to make sure the upgrade playbooks set the `running_upgrade` var at the top. The variable name can be improved :)

Reply:
Hmmm, even if it could work - do you think it is a good solution? 😉

Reply:
What about including the tasks file (`tasks/restart_kafka.yml`) directly, not going through the handler, in the upgrade playbooks? Could that work?

Reply:
Yes thats a good idea! 

Can you update these lines:
https://github.com/confluentinc/cp-ansible/blob/6.0.0-post/upgrade_kafka_broker.yml#L9
https://github.com/confluentinc/cp-ansible/blob/6.0.0-post/upgrade_kafka_broker.yml#L222

They can be:
```
  handlers:
    - name: restart kafka
      include_tasks: roles/confluent.kafka_broker/tasks/restart_kafka.yml
```

Then this would all work! I'll want to run some upgrade tests

Reply:
@domenicbove Nice! I will do that tomorrow. It is almost midnight here in Norway.... 😀

Reply:
@domenicbove I have now updated the handlers in the upgrade playbooks as agreed and ready for a another review (and som upgrade tests)!